moveit_core/kinematic_constraints/src/kinematic_constraint.cpp

```diff
@@ -848,25 +851,26 @@ shapes::Mesh* VisibilityConstraint::getVisibilityCone(const robot_state::RobotSt
 }
 
 void VisibilityConstraint::getMarkers(const robot_state::RobotState& state,
-                                      visualization_msgs::MarkerArray& markers) const
+                                      visualization_msgs::msg::MarkerArray& markers)
```

Comment:
`getMarkers()` shouldn't be const. Did you change it because of `clock_ros_`? In this case I would suggest creating a local instance.

Reply:
Even better, just use `rclcpp::Time::now(RCL_ROS_TIME);`

Reply:
Reverted back here https://github.com/AcutronicRobotics/moveit2/commit/beb3bcaa1ca4de4d175018102e7014dbe9bb9593